framework/src/media/audio/audio_manager.c

```diff
@@ -86,6 +86,12 @@ enum audio_card_type_e {
 	OUTPUT
 };
 
+enum audio_card_operation_e {
+	ENABLE = 0,
```

Comment:
How about add prfix
ex) AUDIO_CARD_ENABLE

Reply:
That's good. Modified.